### ISSUE TITLE
Corrupted instalation, can't reinstall

### ISSUE THREAD

**Issue by marcelo-vera** [GoogleCloudPlatform/cloud-code-vscode#651]
**Environment**:
*(Hint: "Report Extension Issue on Github" command will fill these out for you.)*

**Version information**

Cloud Code Extension version: 1.20.1

VSCode version:

OS: W10 pro

Cloud SDK:

Skaffold: 

Kubectl:

**Description**:

**Repro step**:
![cloud_code](https://user-images.githubusercontent.com/67352305/190059099-0f3bd110-d702-4571-8bd0-8c8c0b839adb.png)

**Comment by ChaseMor** [comment]
Could you provide a bit more details about your dev environment in which this occurred (e.g. WSL, or Remote containers, etc.)

**Comment by sbland** [comment]
I got this issue as I opened multiple vscode windows at the same time. Each window attempted to install cloud sdk at the same time causing high CPU and Memory usage. This crashed my machine and next time I loaded up vscode I got the error above. 
I found 2 temporary solutions...
1. Open a single vscode window and wait for it to finish. Note I had to do this at every session start.
2. set the following setting cloudcode.autoDependencies to off
